### PR TITLE
[ParticleMechanics] Python modules init

### PR DESCRIPTION
**Pull request by bodhinandach**
Following #3217 and https://github.com/KratosMultiphysics/Kratos/wiki/Applications-as-python-modules for ParticleMechanics

### CODE REVIEW

File: applications/ParticleMechanicsApplication/python_scripts/apply_mpm_slip_boundary_process.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import KratosMultiphysics
-import KratosMultiphysics.ParticleMechanicsApplication
+import KratosMultiphysics.ParticleMechanicsApplication as KratosParticle
```
Comment (philbucher) [inline applications/ParticleMechanicsApplication/python_scripts/apply_mpm_slip_boundary_process.py]: why do you change this? This is not related to the python modules

File: applications/ParticleMechanicsApplication/python_scripts/kratos_main_particle.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
 
 import KratosMultiphysics
-import KratosMultiphysics.ParticleMechanicsApplication
+import KratosMultiphysics.ParticleMechanicsApplication as KratosParticle
 from particle_mechanics_analysis import ParticleMechanicsAnalysis
```
Comment (philbucher) [inline applications/ParticleMechanicsApplication/python_scripts/kratos_main_particle.py]:
```suggestion
 from KratosMultiphysics.ParticleMechanicsApplication.particle_mechanics_analysis import ParticleMechanicsAnalysis
```
this is what became possible now with the python-modules

File: applications/ParticleMechanicsApplication/python_scripts/apply_mpm_slip_boundary_process.py (hunk 1 above)
Comment (bodhinandach) [reply applications/ParticleMechanicsApplication/python_scripts/apply_mpm_slip_boundary_process.py]: This is just a minor cleanup, to make all the import called as KratosParticle. Not related yes.

Comment (philbucher) [reply applications/ParticleMechanicsApplication/python_scripts/apply_mpm_slip_boundary_process.py]: I checked the file, you are not using anything from the ParticlesApp there, therfore I recommend removing them (also codacy complains abt it being unused which is true)

I recommend to only import them if you need them

Comment (philbucher) [reply applications/ParticleMechanicsApplication/python_scripts/apply_mpm_slip_boundary_process.py]: we can discuss in person if you have questions

Comment (bodhinandach) [reply applications/ParticleMechanicsApplication/python_scripts/apply_mpm_slip_boundary_process.py]: I agree with you. In fact, I removed it from some other python files too. Thanks for the suggestion!

Comment (philbucher) [reply applications/ParticleMechanicsApplication/python_scripts/apply_mpm_slip_boundary_process.py]: It is not correct yet, I will explain it to you in person tmr
Nothing big, don't worry

**Comment by bodhinandach** [comment]
@philbucher can you recheck?

File: applications/ParticleMechanicsApplication/python_scripts/particle_mechanics_analysis.py [hunk 3]
```diff
@@ -13,6 +12,7 @@
 
 # Importing the base class
 from analysis_stage import AnalysisStage
```
Comment (philbucher) [inline applications/ParticleMechanicsApplication/python_scripts/particle_mechanics_analysis.py:14]:
```suggestion
 from KratosMultiphysics.analysis_stage import AnalysisStage
```

**Comment by philbucher** [comment]
One last thing then its ok

If you find some more then you can fix them in later PRs

**Comment by bodhinandach** [comment]
Should I change all the import from KratosMultiphysics similarly, or just the analysis stage is fine?

**Comment by philbucher** [comment]
> Should I change all the import from KratosMultiphysics similarly, or just the analysis stage is fine?

eventually yes, but for now it is ok
you don't have to do it now

**Review by philbucher — APPROVED** [review]
